### DISCUSSION TITLE
404/not-found in root app directory in Next 13

### DISCUSSION THREAD

**Discussion by wes337** [vercel/next.js#43850] [Help]
Even though I have created a `/app/not-found.jsx` component, when going to `mywebsite.com/a-page-that-does-not-exist`, I get the default 404 page: 
![image](https://user-images.githubusercontent.com/9464690/206477225-2082f320-911a-440a-8687-3ebaebbbdb26.png)

The `not-found.jsx` component is working as expected when nested inside subdirs (for example `/app/posts/not-found.jsx`) and when using the `notFound()` function.

Is there anyway to override the styles of the default/generic 404/not found page?

**Comment by cjdunteman** [comment]
They are currently working on a 404.js to handle global errors. See [here](https://beta.nextjs.org/docs/upgrade-guide#migrating-from-pages-to-app)
